tests/all/debug/lldb.rs

```diff
@@ -63,10 +63,6 @@ fn check_lldb_output(output: &str, directives: &str) -> Result<()> {
 
 #[test]
 #[ignore]
-#[cfg(all(
-    any(target_os = "linux", target_os = "macos"),
```

Comment:
Since these tests are `#[ignore]`d by default, and run fine on Windows when explicitly requested, it seems there is no reason to have these guards anymore.